From: Philipp Hug <philipp@hug.cx>
Date: Tue, 20 Mar 2012 12:00:33 +0100
Subject: disable kernel header checking

---
 drbd/Makefile |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drbd/Makefile b/drbd/Makefile
index 0985340..49607ef 100644
--- a/drbd/Makefile
+++ b/drbd/Makefile
@@ -91,9 +91,9 @@ else
   endif
 
   KDIR_Makefile_PATCHLEVEL = $(shell test -e $(KDIR)/Makefile && grep "^PATCHLEVEL = " $(KDIR)/Makefile | cut -d " " -f 3)
-  ifneq ($(findstring $(KDIR_Makefile_PATCHLEVEL),12345),)
-    $(error "won't compile with this kernel version")
-  endif
+#  ifneq ($(findstring $(KDIR_Makefile_PATCHLEVEL),12345),)
+#    $(error "won't compile with this kernel version")
+#  endif
 
   .PHONY: drbd.o default all greeting clean kbuild install dep tags
 
-- 
