From: Philipp Hug <philipp@hug.cx>
Date: Tue, 20 Mar 2012 14:51:19 +0100
Subject: init-interactive-debian-613654

---
 scripts/drbd |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/scripts/drbd b/scripts/drbd
index 2538e8b..eb5538c 100755
--- a/scripts/drbd
+++ b/scripts/drbd
@@ -17,6 +17,7 @@
 # Default-Stop:   0 1 6
 # X-Start-Before: heartbeat corosync
 # X-Stop-After:   heartbeat corosync
+# X-Interactive:  true
 # Short-Description:    Control drbd resources.
 ### END INIT INFO
 
-- 
