Author: Apollon Oikonomopoulos <apoikos@gmail.com>
Description: Add LSB description to the initscript
Last-Update: 2014-01-15
--- a/scripts/drbd
+++ b/scripts/drbd
@@ -19,6 +19,7 @@
 # X-Stop-After:   heartbeat corosync
 # X-Interactive:  true
 # Short-Description:    Control drbd resources.
+# Description:    Control all Distributed Replicated Block Device resources.
 ### END INIT INFO
 
 DEFAULTFILE="/etc/default/drbd"
